README.md

```diff
@@ -63,7 +63,50 @@ pipenv sync --dev
 pipenv run pytest
 ```
 
-#### Black code formatting integration
```

Comment:
Moved black code formatting section to the bottom of the README to make the steps to run connect more visible.